content/millennium/r4/base/management/encounter.md

```diff
@@ -183,6 +183,8 @@ _Implementation Notes_
 
 * The [Encounter.hospitalization.destination] will be returned as a reference to a [contained] location resource.
 * The [Encounter.location.location] may be returned as a reference to a [contained] location resource.
+* Only one [Encounter.location.location] reference is permitted.
```

Comment:
I don't think we need these implementation notes related to the Retrieve by id operation.

Reply:
whoops that shouldn't be there, removed https://github.com/cerner/fhir.cerner.com/pull/803/commits/1d0f2634bd131127317a63f5458e2f27fc579829#diff-3d460d456f4275cf030a4b8eb00128a8f017087dd7a2918741e96202c45e6c52L186